packages/grafana-runtime/src/config.ts

```diff
@@ -67,6 +67,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   };
   licenseInfo: LicenseInfo = {} as LicenseInfo;
   rendererAvailable = false;
+  rendererVersion = '';
```

Comment:
I would probably suggest to let the backend decide whether CSV files are exported and put that config here. Would make the frontend easier, not having to parse the version over and over again. WDYT?

Looks like the code would only return a rendererVersion if running the service and not the plugin. With a boolean  flag telling if CSV files are supported or a map of features supported when running as a plugin those would always be true/provided since backend plugin enforces only running plugins supported on specific Grafana version.

Reply:
It's a good idea and it would help for the frontend part but I was thinking that the version could be useful for other improvements as well, as showing the `grafana-image-renderer` version somewhere in the UI and an "update is available" message like for Grafana itself. But maybe it should be handled as two different things?

For the plugin, the version is retrieved here: https://github.com/grafana/grafana/pull/34964/files#diff-d4186b06f22ace16bca28532eab61b1aa151af1750803ffb07ab654c077d6162R112

> when running as a plugin those would always be true/provided since backend plugin enforces only running plugins supported on specific Grafana version

I'm not sure I understand what you mean here. I was able to run `grafana-image-renderer:2.1.1` with Grafana 8.0.0-beta3 without any issue, even if v8 is not in the list of dependencies: https://grafana.com/grafana/plugins/grafana-image-renderer/ But maybe it's because it's a beta version?